### PR TITLE
Enhancement: Enable binary_operator_spaces fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] enables and configures the `binary_operator_spaces` fixer
* [x] runs `make cs`

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/v2.8.0#usage:

>**binary_operator_spaces** [`@Symfony`]
>
>Binary operators should be surrounded by space as configured.
>
>Configuration options:
>
>* `align_double_arrow` (`false`, `null`, `true`): (deprecated) Whether to apply, remove or ignore double arrows alignment; defaults to `false`
>* `align_equals` (`false`, `null`, `true`): (deprecated) Whether to apply, remove or ignore equals alignment; defaults to `false`
> * `default` (`'align'`, `'align_single_space'`, `'align_single_space_minimal'`, `'single_space'`, `null`): default fix strategy; defaults to `'single_space'`
> * `operators` (`array`): dictionary of binary operator => fix strategy values that differ from the default strategy; defaults to `[]`

### CODE REVIEW

File: classes/Domain/Entity/TalkTag.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ public static function relations(\Spot\MapperInterface $mapper, \Spot\EntityInte
     {
         return [
             'talk' => $mapper->belongsTo($entity, \OpenCFP\Domain\Entity\Talk::class, 'talk_id'),
-            'tag'  => $mapper->belongsTo($entity, \OpenCFP\Domain\Entity\Tag::class, 'tag_id'),
```
Comment (localheinz) [inline classes/Domain/Entity/TalkTag.php]: @chartjes 

Depending on what the preference is, we can also configure to align `=>` and `=` - what would you prefer?

Comment (chartjes) [reply classes/Domain/Entity/TalkTag.php]: I think I like it all being lined up. 

I also think we should add some notes to the README for developers to let them know they should be running phpcs before every PR they submit.

**Comment by localheinz** [comment]
@chartjes 

Not sure, is that it?

File: classes/Domain/Entity/TalkTag.php (hunk 1 above)
Comment (localheinz) [reply classes/Domain/Entity/TalkTag.php]: We have a note right here: https://github.com/opencfp/opencfp/blob/master/CONTRIBUTING.md#fixing-code-style-issues.

**Comment by localheinz** [comment]
Thank you, @chartjes!

**Pull request merged**